pep-9999.rst

```diff
@@ -993,14 +995,16 @@ PEPs
 .. [#pep612] PEP 612 -- Parameter Specification Variables, Mendoza (https://www.python.org/dev/peps/pep-0612)
 .. [#pep3107] PEP 3107 -- Function Annotations, Winter and Lownds (https://www.python.org/dev/peps/pep-3107)
 
-Type Checkers
--------------
+Type Checkers and IDEs
+----------------------
 
 .. [#mypy] mypy -- Optional Static Typing for Python (http://www.mypy-lang.org/)
 .. [#pycharm] PyCharm -- The Python IDE for Professional Developers (https://www.jetbrains.com/pycharm/)
+.. [#pylance] Pylance -- Fast, feature-rich language support for Python (https://github.com/microsoft/pylance-release)
 .. [#pyre] Pyre -- A performant type-checker for Python 3 (https://pyre-check.org/)
 .. [#pyright] Pyright -- Static type checker for Python (https://github.com/microsoft/pyright)
 .. [#pytype] pytype -- A static analyzer for Python code (https://github.com/google/pytype)
+.. [#vscode] Visual Studio Code -- Code Editing. Redefined (https://code.visualstudio.com/)
```

Comment:
Is there a particular extension that's used with VS Code?

Reply:
There is an official Python by Microsoft, but installing it is automatically suggested (and possible with one click) when opening a Python project. Do you think we should mention it separately?

Reply:
And pylance is a separate extension that extends the extension. (I think, I haven't tried it.) Maybe that is something @jakebailey can clear up?

Reply:
Yes, Pylance is an extension for VS Code, so I'd probably mention them together if anything. Pylance hard depends on the Python extension, so you'd get both by installing Pylance. I wouldn't mention the Python extension alone, as it doesn't do type checking except via Pylance (or via linting with mypy or other tools).